### PR TITLE
fastfetch: add page

### PR DESCRIPTION
**Pull request by suyashbhawsar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by suyashbhawsar** [comment]
@kbdharun

File: pages/linux/fastfetch.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# fastfetch
+
+> A neofetch-like tool but, faster & prettier.
+> Modify ~/.config/fastfetch/config.conf for persistent flags.
+> More information: <https://github.com/LinusDierheimer/fastfetch>.
+
+- Display system information as configured in ~/.config/fastfetch/config.conf:
```
Comment (CleanMachine1) [inline pages/linux/fastfetch.md:7]:
```suggestion
- Display system information:
```

File: pages/linux/fastfetch.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# fastfetch
+
+> A neofetch-like tool but, faster & prettier.
+> Modify ~/.config/fastfetch/config.conf for persistent flags.
```
Comment (CleanMachine1) [inline pages/linux/fastfetch.md:4]:
```suggestion
```

File: pages/linux/fastfetch.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+> More information: <https://github.com/LinusDierheimer/fastfetch>.
+
+- Display system information as configured in ~/.config/fastfetch/config.conf:
+
+`fastfetch`
+
+- Set the structure of the fetch:
+
+`fastfetch --structure <structure>`
```
Comment (CleanMachine1) [inline pages/linux/fastfetch.md:13]:
```suggestion
`fastfetch --structure {{structure}}`
```

File: pages/linux/fastfetch.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+- Load a custom config file:
+
+`fastfetch --load-config <file>`
```
Comment (CleanMachine1) [inline pages/linux/fastfetch.md:17]:
```suggestion
`fastfetch --load-config {{path/to/config_file}}`
```

File: pages/linux/fastfetch.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+- Set the logo to use:
+
+`fastfetch --logo`
```
Comment (CleanMachine1) [inline pages/linux/fastfetch.md:21]:
```suggestion
`fastfetch --logo {{logo}}`
```

File: pages/linux/fastfetch.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+- Set the color of the keys:
+
+`fastfetch --color`
```
Comment (CleanMachine1) [inline pages/linux/fastfetch.md:25]: whats the syntax here? is it hex or just words such as "blue"

File: pages/linux/fastfetch.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+- Set the logo to use:
+
+`fastfetch --separator`
```
Comment (CleanMachine1) [inline pages/linux/fastfetch.md:29]: this doesn't make any sense and is a duplicate

File: pages/linux/fastfetch.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+# fastfetch
+
+> A neofetch-like tool but, faster & prettier.
```
Comment (CleanMachine1) [inline pages/linux/fastfetch.md:3]:
```suggestion
> A CLI tool to display information about your operating system, software and hardware.
```

**Review by CleanMachine1 — CHANGES_REQUESTED** [review]

File: pages/linux/fastfetch.md (hunk 6 above)
Comment (suyashbhawsar) [reply pages/linux/fastfetch.md:24]: Yup, it accepts the strings with the color names.

File: pages/linux/fastfetch.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+
+`fastfetch --load-config {{path/to/config_file}}`
+
+- Set the logo to use:
+
+`fastfetch --logo {{logo}}`
+
+- Set the color of the keys:
+
+`fastfetch --color <color>`
```
Comment (CleanMachine1) [inline pages/linux/fastfetch.md:24]:
```suggestion
`fastfetch --color {{blue}}`
```

File: pages/linux/fastfetch.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+
+- Load a custom config file:
+
+`fastfetch --load-config {{path/to/config_file}}`
+
+- Set the logo to use:
+
+`fastfetch --logo {{logo}}`
+
+- Set the color of the keys:
```
Comment (kbdharun) [inline pages/linux/fastfetch.md:22]:
```suggestion
- Set the color of the keys and title:
```

Ref. 
image
image
image
image

**Review by kbdharun — APPROVED** [review]

**Review by owenvoke — APPROVED** [review]
Nice job, thanks.

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
@CleanMachine1 Can you check this PR, The page LGTM, since you have requested changes merging is blocked in my side.

**Comment by suyashbhawsar** [comment]
@CleanMachine1

**Pull request merged**